src/Minefield/cell.tests.cpp

```diff
@@ -0,0 +1,17 @@
+#include <gtest/gtest.h>
+#include <Minefield/cell.h>
+namespace cell::tests
+{
+    //TEST(OrnellaSuiteTest, dummy_test)
+    //{
+    //    EXPECT_TRUE(true, true);
+    //}
+
+    TEST(Cell, should_not_declare_a_cell_as_taken_by_default)
```

Comment:
#nice, but #should not write tests for your own code - that's Valen's task. You should write tests for Gus!